frontends/ol-components/src/components/LearningResourceExpanded/InfoSectionV2.tsx

```diff
@@ -376,6 +426,77 @@ const InfoItem = ({ label, Icon, value }: InfoItemProps) => {
   )
 }
 
+const DifferingRunsTable: React.FC<{ resource: LearningResource }> = ({
+  resource,
+}) => {
+  if (!resource.runs) {
+    return null
+  }
+  if (resource.runs.length === 1) {
+    return null
+  }
+  const asTaughtIn = resource ? showStartAnytime(resource) : false
+  const prices = []
+  const deliveryMethods = []
+  const locations = []
+  for (const run of resource.runs) {
+    if (run.prices) {
+      prices.push(run.prices)
+    }
+    if (run.delivery) {
+      deliveryMethods.push(run.delivery)
+    }
+    if (run.location) {
+      locations.push(run.location)
+    }
+  }
+  const distinctPrices = [...new Set(prices.flat())]
+  const distinctDeliveryMethods = [
+    ...new Set(deliveryMethods.flat().map((dm) => dm?.code)),
+  ]
+  const distinctLocations = [...new Set(locations.flat().map((l) => l))]
+  if (
+    distinctPrices.length > 1 ||
+    distinctDeliveryMethods.length > 1 ||
+    distinctLocations.length > 1
+  ) {
+    return (
+      <DifferingRuns data-testid="differing-runs-table">
```

Comment:
Would this work better as an html table? Thinking in terms of Voiceover / accessibility.

Reply:
I just looked into this a little bit. My impression is that real HTML tables are nice because screenreaders read row and column headers pretty logically and support arrow key navigation more naturally. (Example: with VoiceOver, VO = voiceover modifier combo... VO + down = navigate down, and VO reads header for new row but not for column, since it's the same).

But with the current layout of this table, it's hard to actually implement as a real HTML table:

<img width="412" alt="Screenshot 2024-11-07 at 1 19 20 PM" src="https://github.com/user-attachments/assets/edb02dbc-c14d-4194-890b-3c6c127d4202">

We can make it look right by using colspan, but then "Location" loses its association with its row, which defeats the a lot of the purpose of using a real table.

I could be wrong, but I don't see a good way to make this a real table.

Reply:
Also, the current navigation, at least in voiceover, seems pretty fine to me.

VO reads
"November 11, 1250 dollars, in person, location: Cambridge, MA, February 10, ..."

which seems ok.